Controllers/Project/UserController.cs

```diff
@@ -69,9 +69,9 @@ public ActionResult<UserV1> GetCurrentUser()
 
     // PUT / edit user by user id
     [HttpPut]
-    [Route("{userId:int}")] 
+    [Route("current/edit")]
     [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
-    public IActionResult UpdateUser(int userId, UpdateRequest editUser)
+    public ActionResult UpdateUser(int userId, UpdateRequest editUser)
```

Comment:
I think this needs to be: `ActionResult<UserV1>`.

I'm not familiar with this UpdateRequest type though? What is that?

Reply:
From Jason's example (see UpdateRequest in the model).  UpdateRequest allows you to omit the password field.  Otherwise it would be required.  It changes the password request to NULL so that no error is returned without it.